galaxy/Dockerfile

```diff
@@ -65,8 +68,6 @@ RUN mkdir -p $GALAXY_CONFIG_DIR $GALAXY_CONFIG_DIR/web && chown -R $GALAXY_USER:
 RUN su $GALAXY_USER -c "cp $GALAXY_ROOT/config/galaxy.ini.sample $GALAXY_CONFIG_FILE"
 ADD ./reports_wsgi.ini.sample $GALAXY_CONFIG_DIR/reports_wsgi.ini
 ADD ./htpasswd /etc/nginx/htpasswd
-ADD ./servers.ini $GALAXY_CONFIG_DIR/servers.ini
-RUN cat $GALAXY_CONFIG_DIR/servers.ini >> $GALAXY_CONFIG_FILE && rm $GALAXY_CONFIG_DIR/servers.ini
```

Comment:
I don't think the handlers are binding themselves to a web port, they are probably using lib/galaxy/main.py to start a process without a web server.